src/components/Notifications/NotificationsFeed.tsx

```diff
@@ -107,7 +108,11 @@ const NotificationsFeed = React.memo((props: NotificationsFeedProps) => {
   return (
     <>
       <Mobile>
-        <Modal open={isMobile && isOpen} onClose={onClose}>
+        <NotificationFeedModal
+          open={isMobile && isOpen}
+          onClose={onClose}
+          title={i18nTabs[locale].feed.title}
+        >
```

Comment:
I know it comes from before. Out of curiosity, why will the feed modal open if it is already open or on mobile?

Reply:
good catch